### DISCUSSION TITLE
没有办法设置 responseType: "blob" 吗？

### DISCUSSION THREAD

**Discussion by ryanli7426** [alovajs/alova#223] [Q&A] [answered]
responseType: "blob"

**Comment by JOU-amjs** [comment, marked as answer]
这个是需要按你使用的请求适配器而定的，如果使用的是XMLHttpRequest适配器和axios适配器，是可以这样设置的，因为XMLHttpRequest和axios都支持这个参数。
但如果使用的是GlobalFetch，它内部是用fetch API实现的，那你就得通过response.blob()来获取
